scripts/karma/config.js

```diff
@@ -24,11 +24,11 @@ module.exports = {
   // preprocess matching files before serving them to the browser
   // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
   preprocessors: {
-    'test/unit/index.tsx': ['webpack'],
+    'test/unit/index.tsx': ['webpack', 'sourcemap'],
```

Comment:
What does `sourcemap` do?

Reply:
When you get minified or transpiled code (like things generated by ts-loader and babel-loader), its difficult to read when debugging. Sourcemaps provide a way for the browser to show the developer the original code (or at least readable). :)